### DISCUSSION TITLE
How do i install classic yarn

### DISCUSSION THREAD

**Discussion by fliiiix** [yarnpkg/berry#5258] [General]
How does one install classic yarn?

I failed with npm
```
npm install --global yarn@v1.22.19
```

asdf 

```
home/l33tname/.asdf/installs/yarn/1.22.19/bin/yarn --version
3.3.0
```

and even the rpm package

reasons is GitLab is still using `--pure-lockfile` which is not supported on the newer versions.

**Comment by merceyz** [comment]
You most likely ran `yarn set version ...` in either the project itself or one of its parent directories and it's overriding your default version.
You should be able to run `yarn set version classic` in your project to make it use v1.
